src/query/service/src/interpreters/interpreter_unsetting.rs

```diff
@@ -46,17 +49,44 @@ impl Interpreter for UnSettingInterpreter {
         let mut keys: Vec<String> = vec![];
         let mut values: Vec<String> = vec![];
         let mut is_globals: Vec<bool> = vec![];
+        let settings = self.ctx.get_settings();
         for var in plan.vars {
             let (ok, value) = match var.to_lowercase().as_str() {
                 // To be compatible with some drivers
                 "sql_mode" | "autocommit" => (false, String::from("")),
+                "max_threads" => {
```

Comment:
There are duplicate codes to handle the default value for specific setting, it makes the codes hard to maintain.

Why not use `let default_val = settings.check_and_get_default_value(setting)?.to_string();` instead ?

Reply:
If so, I think the setting `max_memory_usage` can also handle as same now.